core/platform/auth/firebase_auth_services_test.py

```diff
@@ -153,19 +161,19 @@ def _update_user(self, uid, disabled=False):
         """
         if uid not in self._users:
             raise firebase_admin.auth.UserNotFoundError('%s not found' % uid)
-        self._set_user(uid, disabled)
+        self._set_user_fragile(uid, disabled)
 
-    def _set_user(self, uid, disabled):
+    def _set_user_fragile(self, uid, disabled):
         """Sets the given properties for the corresponding user.
 
+        FRAGILE! The dict keys used in the UserRecord constructor is an
```

Comment:
is --> are (you're talking about "the dict keys")

Reply:
Done.